docs/develop/add-in-manifests.md

```diff
@@ -119,13 +123,15 @@ The following XML manifest example hosts its main add-in page in the `https://ww
 
 ```XML
 <?xml version="1.0" encoding="UTF-8"?>
-<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xsi:type="TaskPaneApp">
+<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="TaskPaneApp">
+  <!--IMPORTANT! Id must be unique for each add-in. If you copy this manifest ensure that you change this id to your own GUID -->
```

Comment:
Consider changing "If you copy this manifest ensure that you change this id to your own GUID" to something more direct. Like "Change this id to the GUID of your add-in." (or however that GUID acquisition should be phrased).